resources/views/checkout/partials/sections/login.blade.php

```diff
@@ -1,34 +1,55 @@
+<login v-slot="{ email, password, go, loginInputChange, emailAvailable, logout }">
     <section>
-        <form id="login" class="grid gap-5 md:grid-cols-2" v-on:submit.prevent="go()">
+        <div class="grid gap-5 md:grid-cols-2">
+            <template v-if="!loggedIn">
+                <x-rapidez-ct::input
+                    label="E-mail address"
+                    name="email"
+                    type="email"
+                    v-bind:value="email"
+                    v-on:input="loginInputChange"
+                    v-on:blur="$root.guestEmail = email; if(!password) { go() }"
+                    required
+                    placeholder="Enter your e-mail address"
+                />
 
-            <x-rapidez-ct::input
-                label="E-mailaddress"
-                name="email"
-                type="email"
-                v-bind:value="email"
-                v-on:input="loginInputChange"
-                required
-                placeholder="Enter your e-mailaddres"
-            />
+                <x-rapidez-ct::input
+                    v-if="!emailAvailable"
+                    label="Password"
+                    name="password"
+                    type="password"
+                    ref="password"
+                    v-on:input="loginInputChange"
+                    required
+                />
 
-            <x-rapidez-ct::input
-                v-if="!emailAvailable"
-                label="Password"
-                name="password"
-                type="password"
-                ref="password"
-                v-on:input="loginInputChange"
-                required
-            />
+                <p v-if="!emailAvailable" class="self-end">
+                    @lang('You already have an account with this e-mail address. Please log in to continue.')
+                </p>
+                <p v-else class="self-end">
+                    @lang('We will send your order confirmation to this e-mail address. We will also check if you already have an account so you can checkout more efficiently.')
+                </p>
 
-            <p v-if="!emailAvailable" class="self-end">Je hebt al een account met dit e-mailadres. Login om verder te gaan</p>
-            <p else class="self-end">Naar dit e-mailadres sturen wij uw bestelling bevestiging. Ook controleren we of u al een account hebt zodat u sneller kunt afrekenen.</p>
-
-            <x-rapidez-ct::button.accent v-if="!emailAvailable" type="submit" dusk="continue">
-                @lang('Login')
-            </x-rapidez-ct::button.accent>
-        </form>
+                <x-rapidez-ct::button.accent v-if="!emailAvailable" dusk="continue" v-on:click.prevent="go">
+                    @lang('Login')
+                </x-rapidez-ct::button.accent>
+            </template>
+            <template v-else>
+                <div class="h-[52px] px-4 flex items-center rounded border bg-ct-inactive-200">
```

Comment:
can you double check for me, i don't think that i use a ct-inactive-200 in the design, if you want to use the disabled color, i think i've used the border color for that, maybe it's nice to have a disabled tailwind class for the disabled states and give the disabled class the color of the border within the tailwind config